Themes/default/MessageIndex.template.php

```diff
@@ -519,7 +519,7 @@ function template_topic_legend()
 			<p class="floatleft">', !empty($modSettings['enableParticipation']) && $context['user']['is_logged'] ? '
 				<span class="main_icons profile_sm centericon"></span> ' . $txt['participation_caption'] . '<br>' : '', '
 				' . ($modSettings['pollMode'] == '1' ? '<span class="main_icons poll centericon"></span> ' . $txt['poll'] : '') . '<br>
-				<img src="' . $settings['images_url'] . '/post/moved.png" alt="" class="centericon sizefix"> ' . $txt['moved_topic'] . '<br>
+				<span class="main_icons move centericon"></span> ' . $txt['moved_topic'] . '<br>
```

Comment:
I changed this because the locked, sticky legends refer to those that are to the _right_ of the topic. Moved should be referring to the same icon, the topic icon is not the legend, is just a decoration.

Reply:
![Screenshot_36](https://user-images.githubusercontent.com/3318127/151686909-c3e77afb-cc56-4728-9ada-3c603549ff32.png)